app/views/application/_navigation_member_pages.html.haml

```diff
@@ -1,12 +1,5 @@
 %li.menu-item.menu-item-has-children
   = link_to t('menus.nav.members.member_pages'), member_pages_path
 
-  %ul.sub-menu
-
-    %li.menu-item
-      = link_to t('menus.nav.members.shf_meeting_minutes'), shf_documents_path
-
-    - HighVoltage.page_ids.each do |page|
-      - unless page == 'index'
-        %li.menu-item
-          = link_to page.capitalize, page_path(page)
+  = render partial: 'shf_documents/documents_list',
```

Comment:
This lists 1) the "meeting minutes" link (which goes to the index view of the meeting minutes documents), and 2) all of the static pages with links created by the HighVoltage gem.